lib/xclarity_client.rb

```diff
@@ -1,6 +1,8 @@
 require 'xclarity_client/version'
+require 'utils/logger'
 
 module XClarityClient
+  LOG = (XClarityClient::XClarityLogger.new).freeze
```

Comment:
@rodneyhbrown7 we need the `logs/lxca_client.log` to save default logs in case of doen't exists a log named `$lenovo_log` or `$log` in the project which use our client.